### PR TITLE
font moved in body

### PR DESCRIPTION
**Pull request by noidea1**
font moved in body and experimental background color

### CODE REVIEW

File: tests/style.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
 body {
-    background-color: #7DB1FF;
```
Comment (takispig) [inline tests/style.css]: Afto to xroma i kati paraplisio den tha itan pio friendly gia ton episkepti tis selidas se sxesh me to mauro?

**Comment by takispig** [comment]
Ontws kalutera etsi, pio organomena vasika! Apla to mauro xroma apo pisw na allaksoume an den sou arese to galazio! An mporeis na vroume tropo gia na einai oi eikones oraia dipla dipla r...tora tis ekana proxeira, alla ama kaneis to parathuro smikrinsi, san se kinito dld, tha deis oti pane opws na nai! To Bootstrap pou se esteila apo oti diavasa kanei kati tetoia kolpakia apla thelei ligo meleth gia na to katalavoume

**Comment by noidea1** [comment]
oxi r apla t alla3a gia na do pos douleui. oute emena m arese t mavro nomizo prepi n a vroume xroma pou na min einai "solid color" den 3ero oute t galazio oute to mavro m arese. min t kanis merge afto itan mono gia na s di3o oti mporis na valis t font st body epidi t ekana ekini tn stigmi. an ginete t feugoume telios afto to branch 8elo na arxiso diko m branch pou na einai t vasiko m giati tr ta exoume kani ligo skata.imaste 2 atoma k exoume 6 branches

**Comment by noidea1** [comment]
an pas sto deutero commit "to make a point" nmz egra4a t skeptiko m ekini tn stigmi

**Comment by noidea1** [comment]
a kai exoume k problima me ta ellinika. egra4a tr tn eisagogh sta ellinika kai m bgazi ieroglifika otan kano t test.

**Pull request closed without merging**